packages/mobile/src/invite/saga.ts

```diff
@@ -314,7 +316,9 @@ export function* doRedeemInvite(inviteCode: string) {
       tempAccount
     )
     if (tempAccountBalanceWei.isLessThanOrEqualTo(0)) {
-      CeloAnalytics.track(CustomEventNames.redeem_invite_failed, { context: 'Empty invite' })
+      ValoraAnalytics.track(AnalyticsEvents.redeem_invite_failed, {
+        error: 'Escrow account has no balance',
```

Comment:
In this case the invite is empty, not the escrow account. There are two accounts used for invite codes: a temp account to pay for verification and some escrow amount sent after verification

Reply:
Ah, gotcha. I will change the error message.